### PR TITLE
Check release branch

### PR DESCRIPTION
**Pull request by kfly8**
If `release.branch` is set, check if the current branch matches that release branch.
We can use `release.hooks` to check if the release branch is correct, but it's better to notice mistakes before editing Changes.

### CODE REVIEW

**Comment by skaji** [comment]
Can you add document of release.branch to https://github.com/tokuhirom/Minilla#configuration too?

File: lib/Minilla.pm [hunk 1]
```diff
@@ -109,6 +109,10 @@ Checking git's untracked files. If there's untracked files, minilla will abort.
 
 This step will run the `git remote`. If there's no remote, minilla will abort.
 
+=head2 CheckReleaseBranch
+
+Cheking git's current branch is `release.branch` key in minil.toml. If no match, minilla will abort.
```
Comment (syohex) [inline lib/Minilla.pm:114]:
```suggestion
Checking git's current branch is `release.branch` key in minil.toml. If no match, minilla will abort.
```

Comment (kfly8) [reply lib/Minilla.pm]: Thanks for the review. Fixed it: https://github.com/tokuhirom/Minilla/pull/305/commits/aede4f7fcad8628704f84419c619cd50d9325962

**Comment by kfly8** [comment]
@skaji Thanks for the review.  I added document of `release.branch`: https://github.com/tokuhirom/Minilla/pull/305/commits/81dd9a1354f34eb1eba3ccdfc907245d4f371a8b

**Comment by skaji** [comment]
According to https://stackoverflow.com/questions/6245570/how-to-get-the-current-branch-name-in-git,
`git branch --show-current` is only available on git 2.22+.
I think we should use `git rev-parse --abbrev-ref HEAD` instead.

**Comment by kfly8** [comment]
Thanks a lot! Fixed it.

- Ref: 
    - branch --show-current option : v2.22+ : https://github.com/git/git/commit/0ecb1fc7269e15be890870937b8b639c987abd08
    - rev-parse: --abbrev-ref option: v1.63+ : https://github.com/git/git/commit/a45d34691ea624e93863e95706eeb1b1909304f3

**Review by skaji — APPROVED** [review]
@kfly8 Thank you. LGTM.

**Review by syohex — APPROVED** [review]
Looks good

**Pull request merged**